components/eam/src/physics/rrtmg/radsw.F90

```diff
@@ -591,6 +601,10 @@ subroutine rad_rrtmg_sw(lchnk,ncol       ,rrtmg_levs   ,r_state      , &
    fusc(1:Nday,pverp-rrtmg_levs+1:pverp) = swuflxc(1:Nday,rrtmg_levs:1:-1)
    fds (1:Nday,pverp-rrtmg_levs+1:pverp) =  swdflx(1:Nday,rrtmg_levs:1:-1)
    fdsc(1:Nday,pverp-rrtmg_levs+1:pverp) = swdflxc(1:Nday,rrtmg_levs:1:-1)
+   ofus (1:Nday,pverp-rrtmg_levs+1:pverp)=  swuflx(1:Nday,rrtmg_levs:1:-1)
```

Comment:
echoing slack convo => these values need to be initialized so that nighttime columns are zero